kuma/core/helpers.py

```diff
@@ -349,3 +351,17 @@ def number(context, n):
     if n is None:
         return ''
     return format_decimal(n, locale=_babel_locale(_contextual_locale(context)))
+
+
+@register.function
+@jinja2.contextfunction
+def cache_bust(context, path):
+    """Return the versioned filename of an asset in media/, given its
+    non-versioned filename."""
+    with open('media/rev-manifest.json') as manifest_data:
+        manifest = json.loads(manifest_data.read())
+
+        try:
+            return static(settings.MEDIA_URL + manifest[path])
```

Comment:
uh, that doesn't make sense, the `MEDIA_URL` settig shouldn't be used with the static template tag as that just does a URL join with `STATIC_URL`. `MEDIA_URL` is `/media/` by default so this will be effectively doing:

urljoin('/media/' + path_to_file)

Also, static files should *not* be built into the MEDIA_ROOT (and accessed via the MEDIA_URL setting). Use STATIC_ROOT and STATIC_URL instead.